src/components/placeholderImage.js

```diff
@@ -3,27 +3,27 @@ import React from "react";
 export const PlaceHolderImage = ({ width = 80, height = 80 }) => (
```

Comment:
Instead of adding the SVG code inside a React component you could import it as follows:

```
import { ReactComponent as Logo } from './logo.svg';

function App() {
  return (
    <div>
      {/* Logo is an actual React component */}
      <Logo />
    </div>
  );
}
```